nano/node/rpc.cpp

```diff
@@ -1963,6 +1964,15 @@ void nano::rpc_handler::account_history ()
 		if (!ec)
 		{
 			hash = node.ledger.latest (transaction, account);
+			if (reverse)
```

Comment:
You can just use account_info open block

Reply:
Done, good point. Ignore the other comment I made (deleted), was confused about legacy vs state blocks.